ocaml/xapi/xapi_cluster_helpers.ml

```diff
@@ -104,6 +106,11 @@ let with_cluster_operation ~__context ~(self : [`Cluster] API.Ref.t) ~doc ~op
       with _ -> ()
   )
 
+let cluster_health_enabled ~__context =
+  let r = Pool_features.is_enabled ~__context Features.Cluster_health in
+  D.debug "%s cluster health is enabled %B" __FUNCTION__ r ;
```

Comment:
Perhaps worth wording this as `"set to %B"` or something like `r ? "enabled" : "disabled"`  or `r ? "enabled" : "not enabled"` to preserve greppability. `"enabled false"` sounds a bit strange otherwise.